superset/views.py

```diff
@@ -1801,26 +1801,38 @@ def fave_dashboards(self, user_id):
     @expose("/created_dashboards/<user_id>/", methods=['GET'])
     def created_dashboards(self, user_id):
         Dash = models.Dashboard  # noqa
+        Log = models.Log
         qry = (
             db.session.query(
                 Dash,
+                sqla.func.count(),
+                sqla.func.count(Log.user_id.distinct()),
             )
+            .join(Log)
             .filter(
                 sqla.or_(
                     Dash.created_by_fk == user_id,
                     Dash.changed_by_fk == user_id,
                 )
             )
+            .filter(
+                sqla.and_(
+                    Log.dashboard_id == Dash.id,
+                )
+            )
             .order_by(
                 Dash.changed_on.desc()
             )
+            .group_by(Dash.id)
```

Comment:
I'm not sure but it seems like this should be `.group_by(Dash)` or individual fields should be specified in the `select` and `group_by` clauses